### DISCUSSION TITLE
域名访问

### DISCUSSION THREAD

**Discussion by YuanYuZh** [1Panel-dev/1Panel#7365] [General]
![1734313981519](https://github.com/user-attachments/assets/04d37409-95cb-4127-97e2-766332aae84d)
提示的命令错误，1pctl reset domain

**Comment by wanghe-fit2cloud** [comment]
![image](https://github.com/user-attachments/assets/a056809c-1d6b-4eb9-ba75-cef5d2e04d3c)

**Comment by YuanYuZh** [comment]
我的意思是web页面提示的命令错了，需要修改

**Reply by wanghe-fit2cloud** [reply]
已修复。
